render/meson.build

```diff
@@ -1,13 +1,27 @@
+renderers = get_option('renderers')
+gbm = dependency('gbm', version: '>=17.1.0', required : false)
+egl = disabler()
+glesv2 = disabler()
+
 wlr_files += files(
 	'allocator.c',
 	'dmabuf.c',
-	'egl.c',
 	'drm_format_set.c',
-	'gbm_allocator.c',
 	'pixel_format.c',
 	'swapchain.c',
 	'wlr_renderer.c',
 	'wlr_texture.c',
 )
+wlr_deps += drm_headers
+
+if gbm.found()
```

Comment:
I'd prefer to split that GBM change into a separate commit, as said elsewhere.

I can do the commit splitting if that sounds complicated to you.